fixtures/tests/modules/submodules/consensus/test_reports.py/test_process_report_data_main_sleep_until_data_submitted.json

```diff
@@ -1,21 +1,265 @@
+      }
+    ],
+    "response": {
+      "jsonrpc": "2.0",
+      "id": 3,
+      "result": "0x6018"
+    }
+  },
   {
     "method": "eth_call",
     "params": [
       {
         "from": "0xF6d4bA61810778fF95BeA0B7DB2F103Dc042C5f7",
+        "maxFeePerGas": "0x83474c5c4",
+        "maxPriorityFeePerGas": "0x1124b53c",
+        "nonce": "0x0",
+        "gas": "0x1e6b8",
         "to": "0x3FD30E8360e2E637be3428fB78A3d8D0ad157197",
         "data": "0xfc7377cd00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000001e000000000000000000000000000000000000000000000000000000000000002400000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000b00000000000000000000000000000000000000000000000000000000000002a0000000000000000000000000000000000000000000000000000000000000000d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000000f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000500000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000700000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000c"
       },
       "latest"
     ],
     "response": {
       "jsonrpc": "2.0",
-      "id": 0,
+      "id": 4,
       "error": {
-        "code": 3,
-        "message": "execution reverted",
-        "data": "0x23dada53"
+        "code": -32000,
+        "message": "insufficient funds for gas * price + value: address 0xF6d4bA61810778fF95BeA0B7DB2F103Dc042C5f7 have 0 want 4390879877938400"
```

Comment:
it doesn't look like it was handled correctly somewhere

Reply:
Fixed